ctapipe/reco/event_classifier.py

```diff
@@ -3,6 +3,8 @@
 from astropy import units as u
 
 from sklearn.ensemble import RandomForestClassifier
+from sklearn.neural_network import MLPClassifier
+from sklearn.preprocessing import StandardScaler
 
```

Comment:
I was playing around with different classifiers and regressors as well but intentionally only imported `RandomForestClassifier` here as a default. I would not add more classifiers here but import them in your own script and call `EventClassifier(classifier=MLPClassifier …)` from there. At least that was my intention and that is how I'm doing it when I play around with other classes...